srcpkgs/busybox/template

```diff
@@ -11,18 +11,22 @@ homepage="http://www.busybox.net"
 distfiles="${homepage}/downloads/busybox-${version}.tar.bz2"
 checksum=3d1d04a4dbd34048f4794815a5c48ebb9eb53c5277e09ffffc060323b95dfbdc
 
-_alternatives_common="
+_alternatives_core="
+ ash:ash:/usr/bin/busybox
  awk:awk:/usr/bin/busybox
  hostname:hostname:/usr/bin/busybox
  sh:sh:/usr/bin/busybox
  vi:vi:/usr/bin/busybox
  logger:logger:/usr/bin/busybox"
 
-alternatives="
- ${_alternatives_common}
+_alternatives="
+ ${_alternatives_core}
  ntpd:ntpd:/usr/bin/busybox
  ntpd:ntpd:/etc/sv/busybox-ntpd"
 
+alternatives="${_alternatives}"
+register_shell="/usr/bin/ash"
```

Comment:
All other shells use `/bin`, I think we should stick to one, even if `/usr/bin` is more correct.

Reply:
Counter point: fish.